modules/dop/services/project/project_package.go

```diff
@@ -551,12 +556,20 @@ func (t *PackageDataDirector) makeGittarRemoteUrl() (string, error) {
 	}
 	logrus.Infof("gittar url %s", gittar)
 
-	members, err := t.bdl.GetMemberByUserAndScope(apistructs.OrgScope, t.Creator.GetContext().UserID, t.Creator.GetContext().OrgID)
+	res, err := t.tokenService.QueryTokens(context.Background(), &tokenpb.QueryTokensRequest{
+		Scope:     string(apistructs.OrgScope),
+		ScopeId:   strconv.FormatUint(t.Creator.GetContext().OrgID, 10),
+		Type:      mysqltokenstore.PAT.String(),
+		CreatorId: t.Creator.GetContext().UserID,
+	})
```

Comment:
make a FIXME comment, this is very dangerous, project package may cause token leak.

Reply:
Added comment.